kedro-org/kedro-viz
# Fix/improve layers labels and watermark highlights

## Description
Fixes #901 

## Development notes
![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/32060364/175041023-0838e180-ec0a-4d1a-8f83-1710b175543a.gif)


## QA notes

I only fix the layer's label, to ensure it gets highlighted when hovering over the layer. 
- I use .5 opacity for the inactive layers name and 1 for the active one, which is slightly different from what you @tynandebold suggested. As I think 0.55 vs 0.85 does not make it look much different but happy to change it back if you prefer 0.55 vs 0.85

For the watermark highlights, waiting for @GabrielComymQB to confirm the colour for both dark and light themes and I'll update them accordingly. 

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/936"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
